tests/phpt/error_handler_handles_fatal_error_only_once.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+Test that fatal errors are only handled once
```

Comment:
It's the exception that can be handled twice, not the fatal error. Please also change the file name to reflect it :pray: